rooms.js

```diff
@@ -1037,6 +1037,9 @@ let BattleRoom = (() => {
 		logData.endType = this.battle.endType;
 		if (!p1rating) logData.ladderError = true;
 		const date = new Date();
+		logData.timestamp = date;
```

Comment:
Yeah, I'm fine with strings. In theory this could introduce TZ issues, but I'd rather have readability. If you care, I'm fine with numeric, though.